docs/configuration/config-file-reference.md

```diff
@@ -82,6 +82,14 @@ api:
   # CLI flag: -http.prometheus-http-prefix
   [prometheus_http_prefix: <string> | default = "/prometheus"]
 
+  # Enable logging of header specific context information
+  # CLI flag: -api.LogHeaders
+  [LogHeaders: <boolean> | default = false]
+
+  # Target Headers for Request logging (if enabled)
```

Comment:
I took me a while to figure out what "Target Headers" really mean. I suggest simple name like "headers_to_log"